generate_diag.sh

```diff
@@ -111,7 +111,7 @@ for i in "$DIAGS_DIR"/${PATTERN}-*.tar.gz; do
 done
 
 if [ -z "$INSIGHTS" ]; then # processing just DSE diagnostic
-    CLUSTER_NAME="$(cat -- */conf/cassandra/cassandra.yaml|grep -e '^cluster_name: '|sed -e "s|^cluster_name:[ ]*\(.*\)\$|\1|"|tr -d "'"|head -n 1|tr ' ' '_')"
+    CLUSTER_NAME=$(cat -- */conf/cassandra/cassandra.yaml|grep -e '^cluster_name: '|sed -e "s|^cluster_name:[ ]*\"\(.*\)\"\$|\1|"|tr -d "'"|head -n 1|tr ' ' '_')
```

Comment:
The regex which was used kept the `"` that surround the cluster name. This ended up generating tarball with names such as `"Test_cluster"-diagnostics.tar.gz` which turned out to be hard to work with.
The change removes the `"` and generate a proper file name now.
Now that I think of it, some folks may not use double quotes to surround their cluster names as yaml conventions don't enforce it.
I'll rework this to use a simple replacement instead of changing the regex.

Reply:
I'm hoping to add DSE support to tlp-cluster shortly so that we can create a DSE specific workflow.

Reply:
regex fixed, I now use `tr` to turn `"` into `_`.